_admin/js/discretionary.js

```diff
@@ -56,7 +56,7 @@ function gotDiscretionaryTickets(jqXHR)
             disc[email] = {};
             calls.push(
                 $.ajax({
-                url: window.profilesUrl+'api/v1/users?$filter=mail eq '+email,
+                url: 'https://profiles.burningflipside.com/api/v1/users?$filter=mail eq '+email,
```

Comment:
I figured you might not like this, but unfortunately the template based rendering stuff doesn't work with that mode. Additionally, I still can't get the docker stuff to actually work on any of my systems so I can't test it. So right now this is what is in the code base and in production until someone who can actually test it can do something else.

Reply:
here we have a javascript file which should not be processed by the templating engine and instead be loaded by the browser directly.. the original line of code should function provided the `window.profilesUrl` var has been set in the page..  

I'm guessing that in your move to template rendering you have removed the block that wrote these vars to the page..
`
      $this->body = $this->body.'<script>
var profilesUrl = \''.$this->profilesUrl.'\'
var loginUrl = \''.$this->loginUrl.'\'
var logoutUrl = \''.$this->logoutUrl.'\'
</script>';
`

by restoring this or writing those vars to the page in another fashion the original line of code above should function as expected..